client/landing/stepper/declarative-flow/hosting-site-creation-flow.ts

```diff
@@ -63,19 +60,6 @@ const hosting: Flow = {
 
 		return { submit };
 	},
-	useSideEffect() {
-		const { setHideFreePlan, setHidePlansFeatureComparison } = useDispatch( ONBOARD_STORE );
-
-		useEffect(
-			() => {
-				setHideFreePlan( true );
```

Comment:
Hi @zaguiin 👋 
Just checking, do we want to have the free plan back on this flow?

Reply:
No, we don't. Did it come back on your side? I don't see it when browsing `/setup/new-hosted-site` 🤔

Reply:
That's great; glad you fixed that! When you start to delete code it is difficult to stop 😄

Reply:
Sorry @zaguiini, does this still hide the free plan? I see `setHideFreePlan( true )` being removed!

Reply:
It does 👍 `planTypes` takes precedence over this prop: https://github.com/Automattic/wp-calypso/pull/76771/files#diff-957ebc95d02ddd8f1ba8103e61c3b31f701ca8a93e8c8b43728aa2b5eadf1d37R95

Reply:
Thanks for clarifying this 👍